pkg/graveler/graveler.go

```diff
@@ -1069,7 +1081,7 @@ func (g *Graveler) CreateBareRepository(ctx context.Context, repositoryID Reposi
 		return nil, err
 	}
 
-	repo := NewRepository(storageNamespace, defaultBranchID)
+	repo := NewRepository(storageNamespace, defaultBranchID, false)
```

Comment:
Need to use readonly or specify in the API that readonly is not relevant to bare repository.